generator/src/lib.rs

```diff
@@ -991,21 +997,14 @@ fn generate_function_pointers<'a>(
                 .as_ref()
                 .expect("Command must have return type");
 
-            let type_in_module = fn_cache.get(name.as_str()).cloned();
+            let define_pfn = !fn_cache.contains(name.as_str());
 
-            if let Some((vendor, ident)) = module {
-                if type_in_module.is_none() {
-                    fn_cache.insert(
-                        name,
-                        (format_ident!("{}", vendor.to_lowercase()), ident.clone()),
-                    );
-                }
+            if define_pfn {
+                fn_cache.insert(name);
```

Comment:
Doesn't this shorten to `let define_pfn = fn_cache.insert(name);`?

Reply:
Nice catch!